EnglishFilter/sections/whitelist_stealth.txt

```diff
@@ -48,6 +48,8 @@
 @@||mijn.kpn.com^$stealth
 ! https://github.com/AdguardTeam/AdguardFilters/issues/71872 [Stealth Mode - Referrer]
 @@||sp.lujiahb.com/play.php$stealth
+! https://github.com/AdguardTeam/AdguardFilters/issues/73815 [Stealth Mode - Referrer]
+@@||simplex-affiliates.com^$stealth,domain=coinomi.con

```

Comment:
This is not ad. I guess it can be unblocked without `$domain`
Like that(only requi)
`@@||iframe.simplex-affiliates.com/form?uid=$stealth`